docs/changelog.rst

```diff
@@ -5,6 +5,7 @@ Changelog
 * :bug:`2117` Users can now properly dismiss notifications with long tiles, or dismiss all the pending notifications at once.
 * :bug:`2024` Multiple crypto.com csv import debited entries with same timestamp will be handled correctly.
 * :bug:`2135` Users will now properly see the correct accounting settings when creating a profit/loss report.
+* :bug:`2168` Rokti was not able to import IOTA trades from Bitcoin.de.
```

Comment:
Please change this entry to follow the style of other entries. Example:

"Bitcoin.de users will now be able to properly import IOTA trades"